dashboard/test/controllers/lessons_controller_test.rb

```diff
@@ -289,9 +289,7 @@ class LessonsControllerTest < ActionController::TestCase
         }
       }
     }
-
     I18n.backend.store_translations 'en-US', custom_i18n
-    assert_equal @lesson_name, solo_lesson_in_script.localized_name
```

Comment:
Why did you decide to remove this? It looks like it should work just the same––is it tested elsewhere?

Reply:
Oh based on your commit message, it looks like this doesn't work anymore?

Reply:
nice catch! the 1st commit broke this but the 3rd commit fixed it 🤷‍♂️